### PR TITLE
chore(challenges-utils): Typescript migration of Challenges util test

### PR DESCRIPTION
**Pull request by aleksaKis**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Is part of #43778

### CODE REVIEW

File: client/src/templates/Challenges/utils/index.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import envData from '../../../../../config/env.json';
 const { forumLocation } = envData;
 
 interface GuideData {
-  forumTopicId: number;
+  forumTopicId?: number;
```
Comment (aleksaKis) [inline client/src/templates/Challenges/utils/index.ts:6]: Had to make `forumTopicId` property optional in order for test case `should use search endpoint when no forumTopicId is supplied` to pass, that's one thing that slipped out from us in last PR. Good thing that is covered with unit test :)

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Review by raisedadead — APPROVED** [review]

File: client/src/templates/Challenges/utils/index.ts (hunk 1 above)
Comment (raisedadead) [reply client/src/templates/Challenges/utils/index.ts:6]: Do you want us to resolve the conversation, looks like you intended to leave this note just as a comment?

Comment (aleksaKis) [reply client/src/templates/Challenges/utils/index.ts:6]: Sure, you can resolve it, I just wanted to leave a note

**Pull request merged**